.husky/pre-push

```diff
@@ -0,0 +1,4 @@
+#!/bin/sh
+. "$(dirname "$0")/_/husky.sh"
+
+npm exec --no -- lint-staged --no-stash
```

Comment:
seems there's a missing `lint-staged.config.cjs` ?

Reply:
Thanks forgot about eslint and prettier, Done